piicatcher/scanner.py

```diff
@@ -65,7 +65,10 @@ def scan(self, text):
 
 class ColumnNameScanner(Scanner):
     def __init__(self, exclude_regex=()):
-        self._exclude_regex = re.compile(exclude_regex[0], re.IGNORECASE)   # [0] cause the option comes as tuple
+        if len(exclude_regex) > 0:
+            self._exclude_regex = re.compile(exclude_regex[0], re.IGNORECASE)   # [0] cause the option comes as tuple
```

Comment:
If you dont make it a list, then you dont have to subscript to the first index

Reply:
this comes from `click` as tuple, it's due to CLI options